learn/wavemaker-release-notes/v11-7-2.md

```diff
@@ -0,0 +1,27 @@
+---
+title: "WaveMaker 11.7.2 - Release date: 3 June 2024"
+id: "v11-7-2"
+sidebar_label: "v11.7.2 - latest"
+---
+
+This release includes enhancements, such as Artifact Version Management, Notifications Response in Project Hub, some bug fixes.
```

Comment:
This release includes enhancements, such as Artifact Version Management, Notifications Response in Project Hub t to improve perforamance of Studio Designer and Project opening. 
some bug fixes.